lessons/Data-Science/AI/ai-training/langs/en-us/pages/human-judgment.adoc

```diff
@@ -0,0 +1,69 @@
+= The Importance of Human Judgment
+
+Open the @starter-file{plagiarism}, click "Run", and then respond to the questions below.
+
+== Determine Closeness
+
+@n Use `cosine-similarity` to compare the student-submitted essay with the wikipedia article. Click "Run". What does the program tell you about the comparison? @fitb{}{@ifsoln{`cosine-similarity(student-essay, wiki-article)` produces an output of ~0.97087...,}}
+
+@fitb{}{@ifsoln{ indicating that the texts are extremely close.}}
+
+@n Did you know that "pachyderm" is a synonym for "elephant"? In the student essay, replace all instances of "elephant" with "pachyderm". Try running the program again, to compare `student-essay` and `wiki-article`. What does the program tell you about the comparison?
+
+@fitb{}{@ifsoln{`cosine-similarity(student-essay, wiki-article)` produces an output of ~0.7961..., meaning that the texts are still}}
+
+@fitb{}{@ifsoln{close enough to raise suspicion.}}
+
+
+== A Different Scenario
+
+A teacher asks students to write an essay on one of three topics: modern art, soccer, or zebras. The teacher feeds the essays to a plagiarism detector to compare the essays to one another. It plots 32 points in a 1023-dimensional space.
+
+@n What does the number 32 tell us? @fitb{}{@ifsoln{The teacher has provided the plagiarism detector with 32 student essays.}}
```

Comment:
I would suggest that the top of this page asks them to describe how many dimensions and points the model for the elephant writing will have an that the class come back together to discuss before they get to this point.

Reply:
I like this as a scaffold, but also think kids will tear out their hair if they need to count how many unique words are in the elephant essay. 

Dorai is setting up a function where students can see the bags of words for different documents. Perhaps they can use that (once it's available) to determine number of dimensions.